### DISCUSSION TITLE
AIrflow Webserver only reading last 100 lines from the pod logs

### DISCUSSION THREAD

**Discussion by nirutgupta** [apache/airflow#21466] [Q&A]
### Apache Airflow version

2.2.3 (latest released)

### What happened

Hi I see when task is in running state, airflow reads pod logs but only last 100 lines.

```
*** Falling back to local log
*** Trying to get logs (last 100 lines) from worker pod XXX ***

```

### What you expected to happen

Is there any configuration variable to increase this number. Why are we just limiting to 100, is there any specific reason to do that? 

### How to reproduce

It is a default behaviour.
I am using kubernetes executor.

https://github.com/apache/airflow/blob/main/airflow/utils/log/file_task_handler.py#L145-L154 these are the code lines

### Operating System

Mac OS 

### Deployment

Official Apache Airflow Helm Chart

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by potiuk** [comment]
Please open Discission https://github.com/apache/airflow/discussions not an issue in such cases (When you created issue in the form, you were several times informed that discussions are better for that).

The reason is performance of retrieving the logs (I believe). No you can't change it currently. I suggest you to write (or wrap) your remote commands to only produce "useful" logs and filter out the "useless" ones - otherwise transmitting your logs is going to take significant of your task execution time (note that during log pulling the POD is running and consumes resources), so my suggestion would be to make sure the task you are running does not produce excessive logs.

If you would like to make it configurable and have good justification for it, You might try to start a Pull Request or creae a feture proposal (but it should have good justification I think because the solution with filtering logs is much more efficient and recommended.
